test/reduced.js

```diff
@@ -18,11 +19,11 @@ describe('reduced', function() {
     // black box test.
     eq(
       R.reduce(
-        function(acc, v) {
+        _xwrap(function(acc, v) {
```

Comment:
Same as above, I'd prefer not to resort to testing on stuff that isn't exported.